### ISSUE TITLE
PSU bug: missing $ on glossary-creation

### ISSUE THREAD

**Issue by EleanorRobson** [oracc/oracc#67]
When the "oracc harvest" command is run, the entries for the second elements of PSUs are created without the necessary $ in the @form line. So for instance, for good old ŠE.DIRI = Addaru arkû the first entry is formed correctly, as:

@entry Addaru [1] MN
@form ŠE.DIRI %akk-x-stdbab $Addaru $(arkû)
@sense MN 1
@end entry

But the second is created as:

@entry warkû [rear] AJ
@form ŠE.DIRI %akk-x-stdbab $(Addaru) arkû
@sense AJ intercalary
@end entry

i.e., without a $ before arkû in the @form line (should be $arkû). If this isn't corrected and one then runs "oracc merge", the "oracc check" command correctly notifies this as an error:

01bld/qpn.glo:1413: during load_acd: bare word in FORM [fi=0]

It's a nuisance to go back and add the $s by hand.

**Comment by EleanorRobson** [comment]
This is going to be of urgent concern when we move to the new server, as it stops the build. The cams/gkab glossaries have 400-odd such error lines in and I really don't want to have to fix them all by hand...

**Comment by stinney** [comment]
No, you don't have to.  When I first installed that diagnostic I wrote a script to fix these.  It will take me a day or two, but I will take care of these.  Maybe I will just run the script on all the glossaries on Oracc--though many have already dealt with this and I think gkab is probably in worse shape than most simply because the lemming was done and dusted years before this error message was introduced 

 Steve

**Comment by EleanorRobson** [comment]
OK, great!

**Comment by EleanorRobson** [comment]
I do need to know accurately when this is going to be fixed, as I now have only two working days (Tue & Thu this week) to finish teaching prep. I don't mind if it's not fixed: I just need to know whether cams/tlab — which I had to abandon last time I was prepping classes 2 years ago — is going to be viable or whether I revert to scans & photocopies.

**Comment by EleanorRobson** [comment]
Ack. I added new texts to cams/tlab, unthinkingly merged the glossaries, and now all the "bare word in FORM" errors are back, which break the build. Is there a work-around?

**Comment by stinney** [comment]
I'll investigate more--I *thought* the actual bad generation on harvest bug was fixed, and that what you are seeing is legacy errors.  This may still be true if each merge is bringing in stuff again from unfixed glossaries like SAA.  More later ...
